.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 Cargo.lock
+*~
+*#
```

Comment:
I think there needs to be a newline here, otherwhise it won't pass travis. (Pretty sure this is because cargo doesn't work yet with emacs, otherwhise newline is always added at the end afaik. We should have a look into the cargo module for emacs to enable it...)